mizar/dp/mizar/workflows/builtins/pods/create.py

```diff
@@ -127,24 +127,36 @@ def run(self):
 
         # Get 'mizar.com/egress-bandwidth' from pod annotations
         egress_bw = int(0)
+        pod_network_class_value = "Premium"
+        pod_network_priority_value = "High"
         if os.getenv('FEATUREGATE_BWQOS', 'false').lower() in ('true', '1'):
+            pod_network_class_value = "BestEffort"
+            pod_network_priority_value = "Medium"
             annotations = self.param.body['metadata'].get('annotations', {})
             if len(annotations) > 0:
-                k8s_egress_bw = annotations.get(CONSTANTS.MIZAR_EGRESS_BW_TAG)
+                mizar_egress_bw = annotations.get(CONSTANTS.MIZAR_EGRESS_BW_TAG)
+                mizar_pod_network_class = annotations.get(CONSTANTS.MIZAR_NETWORK_CLASS_TAG)
+                mizar_pod_network_priority = annotations.get(CONSTANTS.MIZAR_NETWORK_PRIORITY_TAG)
                 # Convert [KB|MB|GB]/s to bytes per second.
-                if k8s_egress_bw is not None:
-                    if k8s_egress_bw.endswith('K'):
+                if mizar_egress_bw is not None:
+                    if mizar_egress_bw.endswith('K'):
                         egress_bw = int(
-                            float(k8s_egress_bw.replace('K', '')) * 1e3)
-                    elif k8s_egress_bw.endswith('M'):
+                            float(mizar_egress_bw.replace('K', '')) * 1e3)
+                    elif mizar_egress_bw.endswith('M'):
                         egress_bw = int(
-                            float(k8s_egress_bw.replace('M', '')) * 1e6)
-                    elif k8s_egress_bw.endswith('G'):
+                            float(mizar_egress_bw.replace('M', '')) * 1e6)
+                    elif mizar_egress_bw.endswith('G'):
                         egress_bw = int(
```

Comment:
Looks to me 1e9 is near int upper range. Shall we do something for overflow when converting type to int?

Reply:
i think the max int value is much bigger. there's lots of room considering we are most commonly talking 10G / 40G at this time.
```
root@ip-192-168-1-144:~/exvbwmizar# python
Python 2.7.17 (default, Feb 27 2021, 15:10:58) 
[GCC 7.5.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> print sys.maxint
9223372036854775807
>>>
```

Reply:
Ok, I checked doc and Python did special handling for int. I was concern the int defined to be max as 2,147,483,647, but Python will change int to long when necessary.